components/StaffView.tsx

```diff
@@ -22,38 +26,52 @@ const filters = [
   "Level 8",
 ];
 
-const StaffView: React.FC<StaffViewProp> = ({ staff }) => (
-  <div className={styles.compContainer}>
-    <div className={styles.leftContainer}>
-      <h1 className={styles.compTitle}>Staff</h1>
-      <div className={styles.compList}>
-        <ul className={styles.scroll}>
-          {staff.map((c) => (
-            <StaffCard
-              key={c.id}
-              firstName={c.firstName}
-              lastName={c.lastName}
-              phone_number={c.phoneNumber}
-              email={c.email}
-            />
+const StaffView: React.FC<StaffViewProp> = ({ staff }) => {
```

Comment:
This component should not take in a prop, since the staff is being called from the StaffView prop, not its parent prop